src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/PaymentMessageProcessor.java

```diff
@@ -53,14 +53,20 @@ public PaymentMessageProcessor(
     public boolean processNextMessage() throws ServiceBusException, InterruptedException {
         IMessage message = messageReceiver.receive();
         if (message != null) {
-            MessageProcessingResult result = process(message);
-            tryFinaliseProcessedMessage(message, result);
+            switch (message.getLabel()) {
+                case "CREATE":
+                    MessageProcessingResult result = processCreateCommand(message);
+                    tryFinaliseProcessedMessage(message, result);
+                    break;
+                default:
+                    deadLetterTheMessage(message, "Unrecognised message type: " + message.getLabel(), null);
```

Comment:
do we want to log error before dead lettering ? `deadLetterTheMessage` method logs it as info may be change that to error ?

Reply:
I assume it should be an error, no idea why we currently log info

Reply:
ok cool not part of this change so all good with this PR